pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -201,6 +201,13 @@ type CephStatus struct {
 	LastChecked    string                       `json:"lastChecked,omitempty"`
 	LastChanged    string                       `json:"lastChanged,omitempty"`
 	PreviousHealth string                       `json:"previousHealth,omitempty"`
+	PgMap          PgMap                        `json:"pgmap,omitempty"`
+}
+
+type PgMap struct {
+	TotalBytes     uint64 `json:"bytes_total"`
```

Comment:
For the json serialization, for consistency let's use camelCasing instead of underscores.
```suggestion
	TotalBytes     uint64 `json:"bytesTotal"`
```

Reply:
Done.